### PR TITLE
DS-1680 Handle user using the back button to change collection during ingest

### PR DESCRIPTION
**Pull request by gmh04**
https://jira.duraspace.org/browse/DS-1680

### CODE REVIEW

**Comment by aschweer** [comment]
I've marked this as 5.2 because the corresponding Jira issue has fix version = 5.2. However, given that this is still marked "work in progress", it may need to be rescheduled.

**Comment by hardyoyo** [comment]
+1 rescheduling

**Comment by tdonohue** [comment]
As noted in the ticket, this code looks like it may continually create new WorkspaceItems each time you hit the back button. Needs testing / verification.

File: dspace-xmlui/src/main/resources/aspects/Submission/submission.js [hunk 1]
```diff
@@ -248,6 +248,16 @@ function submissionControl(collectionHandle, workspaceID, initStepAndPage)
 {
 	//load initial submission information
 	var submissionInfo = getSubmissionInfo(workspaceID);
+	
+    if(!collectionHandle.equals(submissionInfo.getCollectionHandle())){
+        // Handle selected does not match the handle of the submission (this can
+        // happen if user has used the back button to change collection). Create
+        // a new workspace item. 
+        var dso = HandleManager.resolveToObject(getDSContext(), collectionHandle);
+        var workspace = WorkspaceItem.create(getDSContext(), dso, true);
```
Comment (tdonohue) [inline dspace-xmlui/src/main/resources/aspects/Submission/submission.js]: As noted above, this looks like it'd create a new WorkspaceItem each time you hit the backbutton. May need rework.

**Review by tdonohue — CHANGES_REQUESTED** [review]

**Comment by tdonohue** [comment]
Closing as this PR must be recreated on `dspace-6_x` branch. XMLUI has been removed from `master`. See this mailing list notice: https://groups.google.com/d/msg/dspace-devel/aEFqSn46ryM/iPRr_IMLCwAJ

**Pull request closed without merging**